airflow/models/dag.py

```diff
@@ -3081,10 +3081,14 @@ def bulk_write_to_db(
         # reconcile dag-schedule-on-dataset and dag-schedule-on-dataset-alias references
         for dag_id, base_dataset_list in dag_references.items():
             dag_refs_needed = {
-                DagScheduleDatasetReference(dataset_id=stored_datasets[base_dataset.uri].id, dag_id=dag_id)
-                if isinstance(base_dataset, Dataset)
-                else DagScheduleDatasetAliasReference(
-                    alias_id=stored_dataset_aliases[base_dataset.name].id, dag_id=dag_id
+                (
+                    DagScheduleDatasetReference(
+                        dataset_id=stored_datasets[base_dataset.uri].id, dag_id=dag_id
+                    )
+                    if isinstance(base_dataset, Dataset)
+                    else DagScheduleDatasetAliasReference(
+                        alias_id=stored_dataset_aliases[base_dataset.name].id, dag_id=dag_id
+                    )
                 )
```

Comment:
This is just a formatting change right? (Done by the linter?) Just making sure I didn’t miss a functionality change.

Reply:
that's correct.